css/style.css

```diff
@@ -0,0 +1,84 @@
+/**
+ * Place your custom styles here.
+ */
+
+ /*GLOBAL OVERRIDES*/
+@import url('https://fonts.googleapis.com/css?family=Raleway');
+@import url('https://fonts.googleapis.com/css?family=Oswald');
```

Comment:
Done (again local). If each PR needs to be complete unit, then I'm not ready to commit yet - based on the stand-up, I thought the driving factor was to have a commit done today? If it's to have the code ready for sensible deployment then I doubt that should be done until I've logically finished the areas I'm working on... Again, am new to this, so we should probably discuss your expectations at the next standup.